### PR TITLE
Readme 1time

### PR DESCRIPTION
**Pull request by sameera2004**
Updated 1 time correlation readme

### CODE REVIEW

File: demos/1_time_correlation/readme.md [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
-    demonstrates using the one-time correlation functions in [scikit-xray] 
-    (https://github.com/Nikea/scikit-xray)
+1. [Multi_tau_one_time_correlation_example.ipynb] (https://github.com/scikit-xray/scikit-xray-examples/blob/master/demos/1_time_correlation/Multi_tau_one_time_correlation_example.ipynb)
+    is an example notebook that demonstrates using the one-time correlation functions in [scikit-xray]
+    (https://github.com/scikit-xray/scikit-xray)
 1. ``100_500_NIPA_GEL.npy`` is a pickled numpy dataset from @afluerasu that 
    is used in the 1-time correlation demo
+1. [XPCS fitting] (https://github.com/scikit-xray/scikit-xray-examples/blob/master/demos/1_time_correlation/XPCS_fitting_with_lmfit.ipynb)
+    is an example notebook that demonstrates XPCS data which are fitted with intermediate scattering factor model function
+    in scikit-xray(https://github.com/scikit-xray/scikit-xray) using lmfit Model(http://lmfit.github.io/lmfit-py/model.html).
```
Comment (ericdill) [inline demos/1_time_correlation/readme.md]: can you turn 

- scikit-xray(https://github.com/scikit-xray/scikit-xray)
- lmfit Model(http://lmfit.github.io/lmfit-py/model.html)

into proper links with the [] () syntax?

File: demos/speckle/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+Speckle Analysis and X-ray Speckle Visibility Spectroscopy(XSVS)
+===============================================================
+
+Included files
+--------------
+1. [speckle-plotting.ipynb] (https://github.com/scikit-xray/scikit-xray-examples/blob/master/demos/speckle/speckle-plotting.ipynb)
+    is an example notebook for demonstrates plotting helper functions for speckle analysis using xray_vision.mpl_plotting.speckle module
+    (https://github.com/Nikea/xray-vision).
```
Comment (ericdill) [inline demos/speckle/readme.md]: Can you turn

- `xray_vision.mpl_plotting.speckle module (https://github.com/Nikea/xray-vision)`

into a proper link with [] () syntax?

File: demos/speckle/readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+1. [X-ray_Speckle_Visibility_Spectroscopy.ipynb](https://github.com/scikit-xray/scikit-xray-examples/blob/master/demos/speckle/X-ray_Speckle_Visibility_Spectroscopy.ipynb)
+    is an example notebook demonstrates XSVS analysis using functions in skxray.core.speckle module (https://github.com/scikit-xray/scikit-xray)
```
Comment (ericdill) [inline demos/speckle/readme.md]: Can you turn 

- `skxray.core.speckle module (https://github.com/scikit-xray/scikit-xray)`

into a link with [] () syntax?

**Comment by ericdill** [comment]
Couple of minor line comments asking for markdown links instead of urls. Other than that, looks good to me, thanks for updating the readme!

**Comment by sameera2004** [comment]
@ericdill Thanks for the comments, I fixed them ready for review

**Comment by ericdill** [comment]
Great, thanks @sameera2004 !

**Comment by sameera2004** [comment]
Thanks @ericdill

**Pull request merged**